action.yml

```diff
@@ -102,36 +74,54 @@ inputs:
     required: false
     default: ''
     description: 'The android target API level.'
-  customParameters:
-    required: false
-    default: ''
-    description: >
-      Custom parameters to configure the build.
-
-      Parameters must start with a hyphen (-) and may be followed by a value (without hyphen).
-      Parameters without a value will be considered booleans (with a value of true).
   sshAgent:
     required: false
     default: ''
     description: 'SSH Agent path to forward to the container'
   gitPrivateToken:
     required: false
     default: ''
-    description: >
-      Github private token to pull from github
+    description: 'Github private token to pull from github'
   chownFilesTo:
     required: false
     default: ''
     description: 'User and optionally group (user or user:group or uid:gid) to give ownership of the resulting build artifacts'
   allowDirtyBuild:
     required: false
     default: ''
-    description: >
-      Allows the branch of the build to be dirty, and still generate the build.
-
-      Note that it is generally bad practice to modify your branch
-      in a CI Pipeline. However there are exceptions where this might
-      be needed. (use with care).
```

Comment:
In my opinion it's useful to indicate bad practice for a parameter so that less experienced developers don't start off on the wrong foot.